tools/run_tests/run_tests.py

```diff
@@ -434,6 +434,10 @@ def _get_tox_envs(self, compiler):
       return ('py27',)
     elif compiler == 'python3.4':
       return ('py34',)
+    elif compiler == 'python3.5':
```

Comment:
you also need to actually use the dockerfile in you just added.

see `def dockerfile_dir(self):`